src/main/java/kitchenpos/application/OrderService.java

```diff
@@ -38,45 +35,32 @@ public OrderService(
     }
 
     @Transactional
-    public Order create(final Order order) {
-        final List<OrderLineItem> orderLineItems = order.getOrderLineItems();
-
-        if (CollectionUtils.isEmpty(orderLineItems)) {
-            throw new IllegalArgumentException();
-        }
-
-        final List<Long> menuIds = orderLineItems.stream()
-                .map(OrderLineItem::getMenuId)
+    public Order create(OrderCreateRequest request) {
+        List<OrderLineItem> orderLineItems = request.getOrderLineItems().stream()
+                .map(orderLineItem -> new OrderLineItem(orderLineItem.getMenuId(), orderLineItem.getQuantity()))
                 .collect(Collectors.toList());
+        validateOrderLineItems(orderLineItems);
```

Comment:
```suggestion
        validateOrderLineItemsSize(orderLineItems);
```
처럼 어떤 검증인지 조금 더 자세히 알려주는건 어떨까요 !?